NETPROGRAM/String_Exercise/Exercise09.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+namespace NETPROGRAM.String_Exercise
+{
+    public class Exercise09
+    {
+
+        public bool IsBarcodeEAN13(string s)
+        {
+
+            return (from index in Enumerable.Range(0, s.Length)
```

Comment:
Bài này chắc dùng for bình thường thôi, lạm dụng linq ở đây làm cho code trở nên rối rắm, khó đọc. Thật ra vẫn có cách dùng linq khác ổn hơn

Reply:
done